Content.Server.Database/Model.cs

```diff
@@ -142,6 +150,24 @@ public enum DbJobPriority
         High = 3
     }
 
+    [Table("trait")]
+    public class Trait
+    {
+        [Column("trait_id")] public int Id { get; set; }
+        public Profile Profile { get; set; } = null!;
+        [Column("profile_id")] public int ProfileId { get; set; }
+
+        [Column("trait_name")] public string TraitName { get; set; } = null!;
+        [Column("setting")] public DbTraitSetting Setting { get; set; }
+    }
+
+    public enum DbTraitSetting
+    {
+        // These enum values HAVE to match the ones in TraitSetting in Content.Shared
+        No = 0,
+        Yes = 1,
+    }
```

Comment:
Is this an efcore thing? Why can't the shared enum be used.

Reply:
i kinda took job preference as a setting, is it wrong?